views/admin/blocks/admin_module_config_form.tpl

```diff
@@ -0,0 +1,5 @@
+[{$smarty.block.parent}]
+
+[{if $oModule->getId() == 'wdoxidee'}]
```

Comment:
If you pass it to the template, you can use `Helper::MODULE_ID` here. Or, even better, we could extend the `Module` module to have a method `isOurs()` or something similar.